src/neo/SmartContract/Native/Tokens/NeoToken.cs

```diff
@@ -89,6 +89,8 @@ private BigInteger CalculateBonus(StoreView snapshot, BigInteger value, uint sta
             return value * sum * NeoHolderRewardRatio / 100 / TotalAmount;
         }
 
+        private bool shouldRefreshCommittee(StoreView snapshot) => snapshot.Height % (ProtocolSettings.Default.CommitteeMembersCount + ProtocolSettings.Default.ValidatorsCount) == 0;
```

Comment:
new epoch?

Reply:
We plan to refresh committee members / validators, grant voter incentive every 28 blocks

Reply:
Why do you need to refresh committee members / validators every 28 blocks?

Reply:
Because if the committee members change during the 28 blocks, some voters will not get incentive even if they have successfully voted for committee members (i.e. an address is the 3rd committee member from block 4 to 14, but it's voter will not get any incentive), and in such cases the committee member itself won'y get any incentive due to current incentive plan.

Reply:
I think it's okay. We just need to be statistically fair.

Reply:
But for current incentive plan there is a chance for big NEO holders to influence others incentive income, which I think is a problem.

Reply:
How?

Reply:
Let's define V(i) equals the number of Neo voting for committee i. Then for example, there exist such a NEO holder, the NEO he holders is more than V(k) - V(k + 1) where 0 <= k <= 19. In block k he sends a vote with all his NEO for the (k+1)th committee member so that the original kth and (k+1)th committee member interchanges. He will get incentive and the voters of original kth committee member will lose theirs. Then in next block he votes for the (k+2)th so that the original (k + 1)th and (k+2)th committee member will interchange. In this block he will still get incentive and the voters of original (k+1)th committee member will lose theirs...... As long as this holder's NEO is more than V(k) - V(k - 1) he will succeed in this block.

This is likely to happen because of the current incentive plan. People fear that they will not be able to get incentive because of committee member changing, so they will tend to vate for "safe" committee members, say the 1st one. The votes of other committee members can be relatively low so that the gap between them is also low enough to make it possible for big NEO holders to play this trick.